docs/development/build-instructions-gn.md

```diff
@@ -90,29 +90,35 @@ $ gclient sync -f
 
 ## Building
 
+There are three different build configurations you can potentially set up - `Debug`, `Testing`, and `Release`.
+
+In your setup, you should replace `OUT_DIR` with your desired build configuration of either `Debug`, `Testing`, and `Release`, and `out_dir.gn` correspondingly with `debug.gn`, `testing.gn` or `release.gn`.
```

Comment:
Maybe instead of `out_dir.gn` use `build_type.gn`
```suggestion
In your setup, you should replace `OUT_DIR` with your desired build configuration of either `Debug`, `Testing`, and `Release`, and `build_type.gn` correspondingly with `debug.gn`, `testing.gn` or `release.gn`.
```